samples/client/petstore/java/microprofile-rest-client/src/main/java/org/openapitools/client/model/Tag.java

```diff
@@ -25,66 +27,85 @@
 import javax.json.bind.annotation.JsonbProperty;
 
 /**
-  * A tag for a pet
- **/
-
-public class Tag  {
-  
+ * A tag for a pet
+ */
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaClientCodegen")
+public class Tag {
   private Long id;
 
   private String name;
-  
- /**
+
+
+  public Tag id(Long id) {
+    
+    this.id = id;
+    return this;
+  }
+
+   /**
    * Get id
    * @return id
   **/
+  @javax.annotation.Nullable
   @JsonbProperty("id")
+
+
   public Long getId() {
     return id;
   }
 
-  /**
-    * Set id
-  **/
+
   public void setId(Long id) {
     this.id = id;
   }
 
-  public Tag id(Long id) {
-    this.id = id;
+
+  public Tag name(String name) {
+    
+    this.name = name;
     return this;
   }
 
- /**
+   /**
    * Get name
    * @return name
   **/
+  @javax.annotation.Nullable
   @JsonbProperty("name")
+
+
   public String getName() {
     return name;
   }
 
-  /**
-    * Set name
-  **/
+
   public void setName(String name) {
     this.name = name;
   }
 
-  public Tag name(String name) {
-    this.name = name;
-    return this;
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
This also belongs in its own PR, only adds chaos to this review that's long already